docs/GlobalAPI.md

```diff
@@ -233,7 +257,13 @@ Use `describe.each` if you keep duplicating the same test suites with different
 #### 1. `describe.each(table)(name, fn, timeout)`
 
 - `table`: `Array` of Arrays with the arguments that are passed into the `fn` for each row.
```

Comment:
Also here it could stay inline:

```suggestion
- `table`: `Array` of Arrays with the arguments that are passed into the `fn` for each row. If you pass in a 1D array of primitives, internally it will be mapped to a table i.e. `[1, 2, 3] -> [[1], [2], [3]]`.
```